index.html

```diff
@@ -164,14 +363,121 @@
             </div>
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
+            <h2 class="why-choose-us-head">why choose us</h2>
+            <p class="why-choose-us-text">Problems trying to resolve the conflict between the two major realms of Classical physics: Newtonian mechanics</p>
+        </div>
+        <div class="grid-cards">
+            <div class="card">
+                <div class="card-header">
+                <img class="card-icon" src="img/icons/briefcase.svg" alt="briefcase icon">
+                <h5 class="card-header-text">Digital Marketing</h5>
+                </div>
+                <p class="card-content">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+            <div class="card">
+                <div class="card-header">
+                    <img class="card-icon" src="img/icons/shop.svg" alt="Store-shop icon">
+                    <h5 class="card-header-text">Digital Marketing</h5>
+                    </div>
+                    <p class="card-content">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+            <div class="card">
+                <div class="card-header">
+                <img class="card-icon" src="img/icons/notebook.svg" alt="Notebook icon">
+                    <h5 class="card-header-text">Business Growing</h5>
+                    </div>
+                    <p class="card-content">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+            <div class="card">
+                <div class="card-header">
+                <img class="card-icon" src="img/icons/users.svg" alt="Users icon">
+                    <h5 class="card-header-text">National top 50 firms</h5>
+                    </div>
+                    <p class="card-content">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+            <div class="card">
+                <div class="card-header">
+                <img class="card-icon" src="img/icons/stats.svg" alt="Statistics icon">
+                    <h5 class="card-header-text">National top 50 firms</h5>
+                    </div>
+                    <p class="card-content">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+            <div class="card">
+                <div class="card-header">
+                <img class="card-icon" src="img/icons/award.svg" alt="Award icon">
+                    <h5 class="card-header-text">Web Development</h5>
+                    </div>
+                    <p class="card-content">the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+            </div>
+        </div>
+        </section>
+
+        <section class="footer">
+            <div class="footer-header">
```

Comment:
Please pay attention that according the design footer content is aligned with the rest of the sections content. 
So I highly recommend you to wrap all footer content into a `container` same as the content of all the pages. 
You've set up padding 195px everywhere, but this is not flexible solution.